src/field_type.cpp

```diff
@@ -82,9 +82,20 @@ void field_type::load( JsonObject &jo, const std::string & )
                   fallback_intensity_level.dangerous );
         optional( jao, was_loaded, "move_cost", intensity_level.move_cost,
                   fallback_intensity_level.move_cost );
+        optional( jao, was_loaded, "light_emitted", intensity_level.light_emitted,
+                  fallback_intensity_level.light_emitted );
+        optional( jao, was_loaded, "translucency", intensity_level.translucency,
+                  fallback_intensity_level.translucency );
         intensity_levels.emplace_back( intensity_level );
     }
     optional( jo, was_loaded, "underwater_age_speedup", underwater_age_speedup, 0_turns );
+    optional( jo, was_loaded, "decay_amount_factor", decay_amount_factor, 0 );
+    optional( jo, was_loaded, "apply_slime_factor", apply_slime_factor, 0 );
+    optional( jo, was_loaded, "dirty_transparency_cache", dirty_transparency_cache, false );
```

Comment:
It seems to me that this `dirty_transparency_cache` bool may be able to be determined automatically at runtime based on whether the field emits light or is translucent.  Determining it automatically would imo be more robust than requiring people modifying/adding new fields to set it correctly.

Reply:
I agree. That is not final state of field type object and I plan to revisit it later. Right now it is just easier to keep flags for all field types in one place in explicit form even if they are somewhat redundant.

Reply:
Makes sense, this jsonization is a pretty massive undertaking already and I'm totally on board with keeping the code simpler for now.